packages/kbn-repo-source-classifier/src/module_id.ts

```diff
@@ -16,4 +16,6 @@ export interface ModuleId {
   repoRel: string;
   /** info about the package the source file is within, in the case the file is found within a package */
   pkgInfo?: PkgInfo;
+  /** path segments of the dirname of this */
+  dirs: string[];
```

Comment:
Where was this new path segments collection was used? Just something useful you remembered to add?

Reply:
Ah, I was using it before I switched to the matcher but I think it's generally useful and no reason not to keep it for now.